ert_shared/cli.py

```diff
@@ -30,12 +35,27 @@ def run_cli(args):
         model, argument = _setup_ensemble_smoother(args)
     elif args.mode == 'es_mda':
         model, argument = _setup_multiple_data_assimilation(args)
+    
     else:
         raise NotImplementedError(
             "Run type not supported {}".format(args.mode))
 
     model.runSimulations(argument)
 
+def _execute_workflow(workflow_name):
+    logging.basicConfig(level=logging.INFO)
```

Comment:
Isn't the logging already initialized on line 15?

Reply:
you are correct in that assessment :+1: